### DISCUSSION TITLE
Xremap is preventing touchpad from turning off when typing

### DISCUSSION THREAD

**Discussion by SayantanRC** [xremap/xremap#599] [Ideas]
**Expected behavior**  
Touchpad should turn off while typing.

**Observed behavior**  
Touchpad keeps working. If xremap is stopped, touchpad stops working while typing, which is the desired behavior.

**OS Information**  
```
[sayantan@nixos:~]$ cat /etc/os-release 
BUG_REPORT_URL="https://github.com/NixOS/nixpkgs/issues"
BUILD_ID="23.11.20231207.b4372c4"
DOCUMENTATION_URL="https://nixos.org/learn.html"
HOME_URL="https://nixos.org/"
ID=nixos
LOGO="nix-snowflake"
NAME=NixOS
PRETTY_NAME="NixOS 23.11 (Tapir)"
SUPPORT_END="2024-06-30"
SUPPORT_URL="https://nixos.org/community.html"
VERSION="23.11 (Tapir)"
VERSION_CODENAME=tapir
VERSION_ID="23.11"
```

Xremap running as flake.

**Comment by k0kubun** [comment]
This reminds me of https://github.com/k0kubun/xremap/issues/152. The issue might not be something handled at the layer of xremap but something lower than that.
